src/components/MapView/index.tsx

```diff
@@ -211,8 +278,43 @@ function MapView({ classes }: MapViewProps) {
         containerStyle={{
           height: '100%',
         }}
-        onClick={() => {
+        onClick={(map: Map, evt: any) => {
```

Comment:
I decided to use the `any` type, since many mapbox handlers use it as type across the project. It would be better to use any to keep consistency. If not, a change is needed for all mapbox event handlers with their respective type. But this should be addressed on another issue